docs/application/web/api/3.0/device_api/tv/index.html

```diff
@@ -1,358 +1,274 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-        <link media="screen" href="./tizen/tizen.css" type="text/css" rel="stylesheet">
-        <title>Tizen TV Web Device APIs Reference</title>
-        <style>
-          table.api-list { width: 90%; margin-left: 2%; }
-          table.api-list tr > td:nth-child(1) { width: 13%; }
-          table.api-list tr > td:nth-child(3) { width: 3em; padding-left: 1em; }
-          table.api-list tr > td:nth-child(4) { width: 5em; }
-          table.api-list tr > td:nth-child(5) { width: 9em; text-align: center; }
-          table.api-list tr > td:nth-child(6) { width: 5em; }
-          table.api-list tr > td:nth-child(7) { width: 9em; text-align: center; }
-          table.api-list tr > td:nth-child(4) .optional { font-style: normal; }
-          table.api-list span.emulated, table.api-list span.partially-emulated { font-weight: bold }
-          /*.emulator-partial { background-color: #ffd0d0; }*/
-          table.api-list .partial-support { font-size: 75%; }
-          table.api-list span.mandatory::after { content: "*"; font-weight: normal; }
-        </style>
-    </head>
-    <body id="content">
-        <div id="title" class="api">
-
-            <h1>Tizen TV Web Device API Reference</h1>
-                    <div class="brief">
-                            <p>The Tizen Web Device API based on JavaScript provides you advanced access to the device&#39;s platform capabilities.</p>
-                            <p>You can develop rich Web applications using the Tizen Web Device API.</p>
-                            <p>The APIs listed in this category are created by the Tizen platform to expose device capabilities to the Web Apps.</p>
-
-            <h4 id="Tizen">Base</h4>
-             <table class="api-list">
-                <thead>
-                <tr><th>API</th><th>DESCRIPTION</th><th>VERSION</th><th>STATE</th></tr>
-                </thead>
-                <tbody>
-                <tr>
-                <td>
-                <a href="tizen/archive.html" data-origin="tizen" data-domain="filesystem" data-category="core" data-status="D" data-version="">
-                        Archive</a></td>
```

Comment:
Question : It looks like "Archive" is removed from the index.html file. Would you please confirm?

Reply:
It was an error. I have added _archive_ to the proper table in the patch set I have just pushed.
Thank you for spotting the problem.